.jenkins/pytorch/build-asan.sh

```diff
@@ -17,5 +17,5 @@ export ASAN_OPTIONS=detect_leaks=0:symbolize=1
 # TODO: Make the ASAN flags a more unified env var
 CC="clang" CXX="clang++" LDSHARED="clang --shared" \
   CFLAGS="-fsanitize=address -shared-libasan" \
-  NO_CUDA=1 DEBUG=1 \
+  USE_CUDA=0 DEBUG=1 \
```

Comment:
Yeah, I just want to point out that passing `NO_CUDA=1` as an env var to the setup.py script is considered part of the externally facing interface, so you'll probably want to keep at least one of these to make sure the BC-layer (which will check `NO_CUDA` envvar if `USE_CUDA` is not set) is working.